lighting_portal_connector/wizard/portal_connector_sync.py

```diff
@@ -133,7 +127,7 @@ def synchronize_products(self, cursor, schema, last_update, reference=None):
                 else:
                     result1_d = {}
                     for k0, v0 in result0_d.items():
-                        v1 = getattr(portal_product, k0, None)
+                        v1 = portal_product[k0]
```

Comment:
In the original line there was an implicit check, if `k0` is not in `portal_product` then return `None`, you lost that in the new line, so it's not protected anymore